library/spdm_responder_lib/libspdm_rsp_csr.c

```diff
@@ -162,17 +164,61 @@ libspdm_return_t libspdm_get_response_csr(libspdm_context_t *spdm_context,
 
     csr_len = *response_size - sizeof(spdm_csr_response_t);
     csr_p = (uint8_t*)(spdm_response + 1);
-    result = libspdm_gen_csr(spdm_context->connection_info.algorithm.base_hash_algo,
-                             spdm_context->connection_info.algorithm.base_asym_algo,
-                             &need_reset, request, request_size,
-                             requester_info, requester_info_length,
-                             opaque_data, opaque_data_length,
-                             &csr_len, csr_p, is_device_cert_model);
+
+    csr_tracking_tag = 0;
+    if (libspdm_get_connection_version(spdm_context) >= SPDM_MESSAGE_VERSION_13) {
+        csr_tracking_tag =
+            (spdm_request->header.param2 & SPDM_GET_CSR_REQUEST_ATTRIBUTES_CSR_TRACKING_TAG_MASK) >>
+            SPDM_GET_CSR_REQUEST_ATTRIBUTES_CSR_TRACKING_TAG_OFFSET;
+
+        if (csr_tracking_tag > SPDM_MAX_CSR_TRACKING_TAG) {
```

Comment:
This check is not needed as `CSRTrackingTag` is 3 bits and its value cannot be larger than 7.

Reply:
Thanks. I have deleted this invalid check.